system/timedatectl.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Shinichi TAMURA (@tmshn)
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+
+DOCUMENTATION = '''
+---
+module: timedatectl
+short_description: Execute `timedatectl` command to set timezone, etc
+description:
+  - Execute `timedatectl` command to set timezone, date/time or enable NTP.
+  - See `man timedatectl` for details.
+version_added: "2.1.0"
+options:
+  ntp:
+    description:
+      - Enable/Disable clock synchronization using NTP.
+      - Default is to keep current setting.
+    required: false
+    choices: [ "yes", "no" ]
+    default: null
+  time:
+    description:
+      - Set current date and/or time of the system clock, specified in the format I(2012-10-30 18:17:16).
+      - Default is to keep current setting.
+    required: false
+    default: null
+  timezone:
+    description:
+      - Set time zone of the system clock, specified in the format I(Asia/Tokyo).
+      - Default is to keep current setting.
+    required: false
+    default: null
+requirements: [ "timedatectl command" ]
+author: "Shinichi TAMURA @tmshn"
+'''
+
+RETURN = '''
+changelogs:
+  description: The change logs concerning to the given arguments.
+  returned: success
+  type: list of dictionary
+  contains:
+    name:
+      description: The name of the column
+      type: string
+    old_value:
+      description: The value before task is executed
+      type: string
+    new_value:
+      description: The value after task is executed
+      type: string
+    command:
+      description: The command executed
+      type: string
```

Comment:
So `command` field is redundant and differences should be shown in a `field` key instead of `old_value`/`new_value` fields, right?  Is there any documents or example about return values?